client/deposits/index.tsx

```diff
@@ -79,6 +97,33 @@ const NextDepositNotice: React.FC = () => {
 	);
 };
 
+const DepositFailureNotice: React.FC = () => {
+	const { hasErroredExternalAccount } = useAccountStatus();
+
+	return hasErroredExternalAccount ? (
+		<BannerNotice
+			status="warning"
+			icon
+			className="deposit-failure-notice"
+			isDismissible={ false }
+		>
+			{ interpolateComponents( {
+				mixedString: __(
+					'Deposits are currently paused because a recent deposit failed. Please {{updateLink}}update your bank account details{{/updateLink}}.',
+					'woocommerce-payments'
+				),
+				components: {
+					updateLink: (
+						<ExternalLink
+							href={ wcpaySettings.accountStatus.accountLink }
```

Comment:
Since we are not using any other field from `wcpaySettings.accountStatus`, I feel accessing `wcpaySettings.accountStatus.accountLink` just where it's used makes it easier to read. Would reconsider if we had multiple fields being used